ipa-core/src/protocol/ipa_prf/prf_sharding/feature_label_dot_product.rs

```diff
@@ -277,30 +281,34 @@ async fn evaluate_per_user_attribution_circuit<FV>(
     ctx_for_row_number: Vec<UpgradedSemiHonestContext<'_, NotSharded, Boolean>>,
     record_id: RecordId,
     rows_for_user: Vec<PrfShardedIpaInputRow<FV>>,
-) -> Result<Vec<Replicated<FV>>, Error>
+) -> Result<Option<Replicated<FV>>, Error>
 where
     FV: SharedValue + CustomArray<Element = Boolean>,
     Replicated<FV>: BooleanArrayMul,
 {
     assert!(!rows_for_user.is_empty());
     if rows_for_user.len() == 1 {
-        return Ok(Vec::new());
+        return Ok(None);
```

Comment:
@akoshelev - I added a user with just one row to the test in order to get code coverage for this line =).